e/erlang/build_info.json

```diff
@@ -0,0 +1,14 @@
+{
+    "package_name": "erlang",
```

Comment:
Lets add package name as `erlang-otp` in dashboard. I see erlang is Org name

Reply:
Okay. Will update it.